### DISCUSSION TITLE
hexagonal architecture

### DISCUSSION THREAD

**Discussion by xx1196** [goravel/goravel#438] [Ideas]
Hello, how can I implement hexagonal architecture and repository in Laravel?

**Comment by hwbrzzl** [comment]
For example?

**Reply by xx1196** [reply]
I don't know, in Laravel PHP I implement hexagonal, but in goravel is different, do u have an example?

**Reply by hwbrzzl** [reply]
How did you implement the hexagonal in Laravel? Could you post your folder structure or something else here?

**Reply by xx1196** [reply]
No, sorry is enterprise project
